lib/tokens/token.js

```diff
@@ -23,6 +23,8 @@
  *
  */
 
+var config = require('../../config').getProperties()
```

Comment:
Initially I tried to pass this in with the other dependencies. But because this module is required from so many different places, that change exploded into a complete mess. I opted for touching less code.